benchexec/tools/legion.py

```diff
@@ -0,0 +1,23 @@
+import benchexec.result as result
```

Comment:
Please add a license header (we strongly prefer Apache 2.0 with a copyright assignment to Dirk, if possible, as this would simplify future project maintenance).